.github/workflows/runtimes-matrix.json

```diff
@@ -84,6 +85,7 @@
     "path": "cumulus/parachains/runtimes/contracts/contracts-rococo",
     "header": "cumulus/file_header.txt",
     "bench_features": "runtime-benchmarks",
+    "bench_flags": "--genesis-builder-policy=none --exclude-pallets=pallet_xcm,pallet_contracts",
```

Comment:
@ggwpez I added removing these excludes to the TODO list here: https://github.com/paritytech/polkadot-sdk/pull/5872